lib/utils/process-jobs.ts

```diff
@@ -227,7 +227,7 @@ export const processJobs = async function(this: Agenda, extraJob: Job) {
 
     // If the 'nextRunAt' time is older than the current time, run the job
     // Otherwise, setTimeout that gets called at the time of 'nextRunAt'
-    if (job.attrs.nextRunAt <= now) {
+    if (!job.attrs.nextRunAt || job.attrs.nextRunAt <= now) {
```

Comment:
not quite sure about this... shouldn't it be 
    `if (job.attrs.nextRunAt && job.attrs.nextRunAt <= now) {
`
to keep the logic the same? "undefined < Date" always returns false

Reply:
There is an "!" at the beginning.

To maintain the old logic the if - condition is.

(job.attrs.nextRunAt == falsly) OR (<= now)